src/freenet/clients/fcp/RequestStatus.java

```diff
@@ -63,10 +69,15 @@ synchronized void restart(boolean started) {
 		this.totalBlocks = total;
 		this.minBlocks = min;
 		this.fetchedBlocks = fetched;
+        // clone() because Date is mutable
+		this.latestSuccess
```

Comment:
>  If you don't want to fix this I can do it after we merge this.

Sorry if I didn't make this clear enough: The problem is not my willingness. I *am* willing to fix such things.
The problem is that I have doubt whether I can justify causing this to cost FPI money by doing it myself: I have tried doing this during plugin-fcp-rewrite, and the amount of time it consumed was almost as much as writing the code took.

You're part of the people who audit the bills, so I would like to ask you to decide now whether I should do this or not. I'll follow your decision :)

For your decision making, more information:

> and one cannot rule out that people set their editors to display whitespace explicitly. (Spoilers: I do!)

Java is not white-space sensitive such as Python. Do we need to support it if people wrongly configure their editors to behave as if it was?